zhenggen-xu/sonic-buildimage
# [device/celestica]: Add support for xcvrd daemon in silverstone

**- What I did**
- Add function to sfputil support xcvrd daemon

**- How I did it**
- Add get_transceiver_change_event function to support xcvrd daemon
- Added a workaround command to make the link go up.

**- How to verify it**
- Verify rsyslog when sfp plug / unplug event occurs.
  - [xcvrd_event.log](https://github.com/zhenggen-xu/sonic-buildimage/files/5458418/xcvrd_event.log)
- Verify port link status when event occurs
  - [xcvr_even_and_port_statust.log](https://github.com/zhenggen-xu/sonic-buildimage/files/5558497/xcvr_even_and_port_statust.log)

- AOI module log : 
  - [AOI-xcvrd.log](https://github.com/zhenggen-xu/sonic-buildimage/files/5812237/AOI-xcvrd.log)
